manual/manual/manual.hva

```diff
@@ -1,4 +1,3 @@
 \input{book.hva}
-\input{fancysection.hva}
```

Comment:
I should have asked before, but why was this include removed? Is it purely doing section styling?

Reply:
Yes, `fancysection.hva` is only doing sections and table of contents styling.